augur/dates/ambiguous_date.py

```diff
@@ -50,6 +50,12 @@ def __init__(self, uncertain_date, fmt, min_max_year=None):
         self.fmt = fmt
         self.min_max_year = min_max_year
 
+    def date_matches_format(self):
+        """Returns a boolean indicating whether the date matches the format."""
+        if re.search(self.regex, self.uncertain_date):
+            return True
+        return False
```

Comment:
_non-blocking_

I'd suggest naming this `matches_fmt` because 1) this is already a method being called on an AmbiguousDate object, so the subject (`date`) is clear, and 2) `fmt` is the name used by the constructor arg and instance property.

(Also, splitting this out from the commit which uses it feels unnecessary to me, given it's a cleanly added new method that's only a few lines long.)